[CUBRIDMAN-25] manual for change column comments without writing column definitions (#226)
CUBRID/cubrid-manual: en/sql/schema/table_stmt.rst

            INHERIT <resolution>, ... |
            AUTO_INCREMENT = <initial_value> |

>             COMMENT [=] 'table_comment_string'

Reviewer: ```suggestion
            COMMENT [=] 'table_comment_string' |
```

            <index_col_name> ::= column_name [(length)] [ASC | DESC]

            <column_comment_definition> ::= column_name [=] 'column_comment_string'

.. note::

>     A column's comment is specified in <*column_definition*> or <*column_comment_definition*>. For <*column_definition*>, see the above CREATE TABLE syntax.

Reviewer: ```suggestion
    A column's comment is specified in <*column_definition*> or <*column_comment_definition*>. For <*column_definition*>, see the above :ref:`CREATE TABLE syntax<column-definition>`.
```

Column's COMMENT
----------------

> A column's comment is specified in <*column_definition*>, which is located at the end of ADD/MODIFY/CHANGE syntax, or in <*column_comment_definition*>, which is located at the end of COMMENT ON COLUMN syntax. To see the meaning of <*column_definition*>, refer to CREATE TABLE syntax on the above.

Reviewer 2: how about?
```suggestion
A column's comment is specified in <*column_definition*> or <*column_comment_definition*>. <*column_definition*> is located at the end of ADD/MODIFY/CHANGE syntax and <*column_comment_definition*> is located at the end of COMMENT ON COLUMN syntax. To see the meaning of <*column_definition*>, refer to CREATE TABLE syntax on the above.
```

> A column's comment is specified in <*column_definition*> or <*column_comment_definition*>. <*column_definition*> is located at the end of ADD/MODIFY/CHANGE syntax and <*column_comment_definition*> is located at the end of COMMENT ON COLUMN syntax. To see the meaning of <*column_definition*>, refer to CREATE TABLE syntax on the above.

Reviewer: ```suggestion
A column's comment is specified in <*column_definition*> or <*column_comment_definition*>. <*column_definition*> is located at the end of ADD/MODIFY/CHANGE syntax and <*column_comment_definition*> is located at the end of COMMENT ON COLUMN syntax. To see the meaning of <*column_definition*>, refer to :ref:`CREATE TABLE syntax<column-definition>` on the above.
```

> In the COMMENT ON COLUMN syntax, column comments are changed by specifying one or more columns.

Reviewer 2: how about?
```suggestion
In the COMMENT ON COLUMN syntax, you can change column comments by specifying one or more columns.
```
